locales/en.default.schema.json

```diff
@@ -1709,6 +1709,87 @@
         "name": "Email signup"
       }
     },
+    "email-signup-banner": {
+      "name": "Email signup banner",
+      "settings": {
+        "paragraph": {
+          "content": "Each email subscription creates a customer account. [Learn more](https://help.shopify.com/en/manual/customers)"
+        },
+        "image": {
+          "label": "Background image"
```

Comment:
@ludoboludo In terms of content, I will tag @Oliviammarcello @gretchen-willenberg - not sure if we should do that since we dont for image with text and image banner and I think it makes sense to keep it consistent

In terms of the heights, I will rebase this off of Ken's banner height PR updates so we can have the most up to date code.

Reply:
I added the same aspect ratio's content that we had on banner as per our meeting today to copy over the settings for banner